package.json

```diff
@@ -17,7 +17,7 @@
     "start": "ng serve",
     "startforelectron": "ng serve --configuration=electronDev",
     "build": "ng build",
-    "buildforproduction": "ng build --source-map=false --configuration=production --base-href /static/web-ui/",
+    "buildforproduction": "ng build --source-map=false --build-optimizer=false --aot --configuration=production --base-href /static/web-ui/",
```

Comment:
Yes, this works if the build-optimizer is `false` however we want is to be activated.